Week-3/session-09/array.js

```diff
@@ -0,0 +1,17 @@
+function divideArray(nums) {
+  let even_nums = []
+  let odd_nums = []
+  for (let num of nums) {
+    if (num%2 === 0){
+    even_nums.push(num)
+    }else{
+    odd_nums.push(num)
+    }
+  }
+  even_nums = even_nums.length == 0 ? 'None' : even_nums.sort()
```

Comment:
this is not the right way to sort , you need to pass call backs

Reply:
done